modules/HomePage/components/TaskContext/TaskContext.types.ts

```diff
@@ -0,0 +1,15 @@
+import { Task } from "./TaskContext";
+
+export interface TasksContextType {
```

Comment:
```suggestion
export interface ITasksContext {
```

No need to write `type` explicitly. Prepending with `I` or `T` should suffice.